### DISCUSSION TITLE
Using the Kelvin Slider

### DISCUSSION THREAD

**Discussion by tqphan** [jaames/iro.js#205] [Q&A] [answered]
I need a Kelvin slider from range 2000-9000.

```
var temperature = new iro.ColorPicker('#temperature', {
  layout: [
    { 
      component: iro.ui.Slider,
      options: {
        sliderType: 'kelvin',
        minTemperature : 2000,
        maxTemperature : 9000,
        layoutDirection: 'horizontal'
      }
    },
  ]
});
```

On event 'input:end' I print to console using:

```
temperature.on('input:end', function(color) {
  console.log(color.kelvin);
});
```

The min and max values I got back were:
2000.2899169921875
8986.129760742188

Am I using the kelvin slider correctly?

**Comment by jaames** [comment, marked as answer]
yes, the conversion between kelvin <-> rgb is just a rough approximation, so it's not extremely accurate and can sometimes give weird values
